WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -249,7 +245,7 @@ - (RemoteBlogSettings *)remoteBlogSettingFromJSONDictionary:(NSDictionary *)json
     //
     if ([[rawSettings numberForKey:BlogRemoteDefaultPostFormatKey] isEqualToNumber:@(0)] ||
         [[rawSettings stringForKey:BlogRemoteDefaultPostFormatKey] isEqualToString:@"0"]) {
-        remoteSettings.defaultPostFormat = PostFormatStandard;
+        remoteSettings.defaultPostFormat = @"standard";
```

Comment:
:+1: since we're already using `BlogRemote*` constants for the keys, we can have some for default values